src/index.js

```diff
@@ -106,8 +110,12 @@ function config(loader, appHost, configModuleId) {
 }
 
 function run() {
-  return ready(window).then(doc => {
-    initialize();
+  return ready().then((isNode: boolean) => {
+    if (!isNode) {
+      initialize();
```

Comment:
1. the global `window` does not exist on node and a TypeError occurs
2. boostrapper imports `aurelia-pal-browser` and that's what it tries to initialize, which is wrong. We have pal-nodejs for that, so under node, just initialize PAL before running the bootstrapper and you're good to go.